src/libasr/intrinsic_func_registry_util_gen.py

```diff
@@ -382,7 +382,13 @@
         {
             "args": [("real",)],
             "return": "int32",
-        },
+        }
```

Comment:
```suggestion
        },
```
We usually put a comma here.